package.json

```
@@ -10,7 +10,8 @@
     "start": "yarn workspace nucleus start",
     "test": "lerna run test --no-private --stream --concurrency 1",
     "build": "lerna run build --parallel --no-private",
-    "lint": "lerna run lint:hbs",
+    "lint": "lerna run lint:hbs && lerna run lint:js",
```

```suggestion
    "lint": "lerna run lint:hbs --parallel && lerna run lint:js --parallel",
```

To improve the run time.